e2e/tests/mcp-external-wizard.spec.ts

```
@@ -0,0 +1,271 @@
+async function fillRegisterServerStep(
+  page: Page,
+  opts: { name: string; prefix: string },
+): Promise<void> {
+  const wizard = page.locator('.kuadrant-mcp-wizard');
+  await wizard.locator('[data-test="mcp-registration-name"]').fill(opts.name);
+  await wizard.locator('[data-test="mcp-registration-namespace"]').selectOption(TEST_NAMESPACE);
+  await wizard.locator('[data-test="mcp-registration-prefix"]').fill(opts.prefix);
+}
+
+async function clickNext(page: Page): Promise<void> {
+  const next = page.locator('.kuadrant-mcp-wizard').getByRole('button', { name: 'Next', exact: true });
```

ESLint

[error] Replace `.locator('.kuadrant-mcp-wizard')` with `⏎····.locator('.kuadrant-mcp-wizard')⏎····`

(prettier/prettier)